packages/web/components/chart/pie-chart.tsx

```diff
@@ -1,10 +1,14 @@
 import type { Options } from "highcharts";
-import dynamic from "next/dynamic";
+import Highcharts from "highcharts";
+import HighchartsExporting from "highcharts/modules/exporting";
+import HighchartsReact from "highcharts-react-official";
 import React, { FunctionComponent, useEffect, useState } from "react";
 
-const HighchartsReact = dynamic(() => import("highcharts-react-official"), {
-  ssr: false,
-});
+// known issue with highcharts and next, fix from their documentation
+// npmjs.com/package/highcharts-react-official#highcharts-with-nextjs
+if (typeof Highcharts === "object") {
+  HighchartsExporting(Highcharts);
+}
```

Comment:
Does the known issue prevent us from dynamically importing HighCharts?

This would help us keep the frontend more performant, especially since HighCharts is only used in a single obscure workflow